frontend/packages/sdk/src/index.ts

```diff
@@ -14,6 +14,7 @@ export * from './pruntime/eip712'
 export { default as createPruntimeClient } from './pruntime/createPruntimeClient'
 export { default as signAndSend, SignAndSendError } from './utils/signAndSend'
 
-export const PhactoryAPI = pruntime_rpc.PhactoryAPI
 export const pruntimeRpc = pruntime_rpc
+export type PhactoryAPI = pruntime_rpc.PhactoryAPI
+export const PhactoryAPI = pruntime_rpc.PhactoryAPI
```

Comment:
What's the purpose of "export `const` PhactoryAPI"?

Reply:
The `export type` statement exports only the type definition, while the `export const PhactoryAPI` statement exports both the type definition and the code implementation.

Reply:
Got it. Isn't there a one-line syntax to reexport the class?